addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -328,7 +322,10 @@ export class Chatter extends Component {
         this.toggleComposer();
         this.state.jumpThreadPresent++;
         // Load new messages to fetch potential new messages from other users (useful due to lack of auto-sync in chatter).
-        this.load(this.state.thread, ["followers", "messages", "suggestedRecipients"]);
+        const requestList = this.requestList;
+        const index = this.requestList.indexOf("attachments");
```

Comment:
This still references attachments in (web_)portal, it shouldn't be here if the feature is not used here.